apps/content-publishing-api/src/controllers/v2/content.controller.v2.ts

```diff
@@ -71,4 +77,15 @@ export class ContentControllerV2 {
     const promises = batchContentDto.batchFiles.map((batchFile) => this.apiService.enqueueBatchRequest(batchFile));
     return Promise.all(promises);
   }
+
+  @Post(':msaId/tombstones')
+  @ApiOperation({ summary: 'Delete DSNP Content for user' })
+  @HttpCode(202)
+  @ApiResponse({ status: '2XX', type: AnnouncementResponseDto })
+  async postTombstone(
```

Comment:
@emresurmeli Could you add a simple test for this in `apps/content-publishing-api/test/app.e2e-spec.ts` (really need to split that up, but another time).

Should be able to mostly just copy/paste the one for the older endpoint

Reply:
Added, emulated the deprecated DELETE endpoint